data/modifications.cfg

```diff
@@ -1,5 +1,35 @@
+#ifdef MULTIPLAYER
+
 [modification]
-    id = "delay_advancements"
-    name = _ "Delay Advancements"
-    type = "hybrid"
+    id=plan_unit_advance
+    name="Plan Unit Advance"
+    description="In multiplayer game, you do not control unit advances if they happen on enemy turn.
+With this add-on, you can set them beforehand:
```

Comment:
These two strings without _,
and the textdomain could be specified in line 1.

add-on -> extension?

Reply:
Updated.